src/main/java/com/microsoft/spring/data/gremlin/telemetry/TelemetryEventData.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.telemetry;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import org.springframework.util.Assert;
+
+import java.time.Instant;
+import java.util.Map;
+
+@Getter
+public class TelemetryEventData {
+
+    private final String name;
+
+    @JsonProperty("iKey")
+    private final String instrumentationKey;
+
+    private final Tags tags = new Tags("Spring-on-azure", "Java-maven-plugin");
+
+    private final EventData data = new EventData("EventData");
```

Comment:
Do we need to make the `EventData` value specific for gremlin instead of `EventData`?

Reply:
fix value from ai json format.